### PR TITLE
Facet view [WIP]

### PR DESCRIPTION
**Pull request by kellrott**
Adding secondary facet based view to view a Grip based graph.

TODO:
- [x] Menu switch between query and facet views
- [x] Paged table view

### CODE REVIEW

**Comment by jordan2lee** [comment]
Needs a requirements.txt file to avoid user to individually install deps

**Comment by kellrott** [comment]
Added requirements.txt

File: README.md [hunk 1]
```diff
@@ -2,10 +2,63 @@
...
 
 
-## Setup 
-To connect server against BMEG.io, get `bmeg_credentials.json` file from [BMEG](https://bmeg.io/analyze/access) 
+# Example server
+
+## Setting up Grip
+
+```
+grip server
```
Comment (jordan2lee) [inline README.md:10]: Returns `Error: File not found: server`

Comment (jordan2lee) [reply README.md:10]: This likely is the reason downstream steps return errors in sections `Load example graph` `Setting up example GDC graph` `Turning on GripQuery web UI`

File: README.md [hunk 2]
```diff
@@ -2,10 +2,63 @@
...
+
+# Connecting to BMEG.io server
+
+## NOTE: grip-query is dependent on updates to Grip 0.7.0, and many functions won't work until bmeg.io is updated
+
+To connect server against BMEG.io, get `bmeg_credentials.json` file from [BMEG](https://bmeg.io/analyze/access)
 
 Start server with command
 ```
 python -m gripquery --port 8060 --grip https://bmeg.io --cred bmeg_credentials.json
```
Comment (jordan2lee) [inline README.md:63]: BMEG app: In GUI, unclear what to type in fields for both views. Possible solutions: 1) display text to explain 2) upon starting app it has loaded one query that will yield results – serves as a template or example 3) dropdown menu has options to select from on the “Facet View”

File: README.md (hunk 1 above)
Comment (kellrott) [reply README.md:10]: What does the command `grip` return?

**Pull request merged**